components/chartmogul/actions/search-customers-via-external-id/search-customers-via-external-id.mjs

```diff
@@ -19,20 +19,24 @@ export default {
   async run({ $ }) {
     const { externalId } = this;
 
-    const items = this.chartmogul.paginate({
-      $,
-      fn: this.chartmogul.listCustomers,
-      params: {
-        external_id: externalId,
-      },
-    });
+    try {
+      const items = this.chartmogul.paginate({
+        $,
+        fn: this.chartmogul.listCustomers,
+        params: {
+          external_id: externalId,
+        },
+      });
 
-    const response = [];
-    for await (const item of items) {
-      response.push(item);
-    }
+      const response = [];
+      for await (const item of items) {
+        response.push(item);
+      }
 
-    $.export("$summary", "Customers Successfully fetched");
-    return response.reverse();
+      $.export("$summary", "Customers Successfully fetched");
+      return response.reverse();
+    } catch (e) {
+      return;
```

Comment:
you should expose the error for better use experience, in this case, you could log the original error log(e); and throw a ConfigurationError with a custom message.